README.md

```diff
@@ -271,25 +270,56 @@ See [these](https://github.com/JuliaRegistries/General/pull/31549#issuecomment-7
 public Julia Language Slack, or directly on the PR itself. This request should include a link to the registration PR and why an expedited merge is needed. In this case, a registry maintainer (see below) can manually review the package (often with stricter standards than the typical package faces, given the lack of bandwidth to review all packages thoroughly).
 
-#### Who can approve an early merge?
+### Who can approve an early merge?
 
 Any registry maintainer may merge a package registration PR early.
 Early merges should still be discussed on the [#pkg-registration](https://julialang.slack.com/archives/C6M4DQA5P) slack channel before merging.
 Registry maintainers are discouraged from merging packages they directly *or* indirectly maintain as mentioned in [CONTRIBUTING.md](https://github.com/JuliaRegistries/General/blob/master/CONTRIBUTING.md#other-ways-to-help).
 
+### When is yanking a release appropriate?
+
+Releases that have already merged can be marked as "yanked" by adding `yanked = true` under the release
+entry in `Versions.toml` ([example](https://github.com/JuliaRegistries/General/pull/94858/files)). This
+tells Pkg to no longer resolve those versions (e.g. when running `Pkg.add()` or `Pkg.resolve()`). However,
+to maintain perfect reproducibility of environments specified with a full manifest, yanked versions that
+are specifically listed in a manifest will continue to instantiate when running `Pkg.instantiate()` and
+yanked versions cannot be re-registered in General with different source code.
+
+If a release is very broken (e.g. contains security vulnerabilities or bugs that are significantly worse 
+than the package erroring on load), then it may be yanked. Releasing an ordinary patch release—potentially 
+reusing the exact same source code as a previous release without the major bug—is a faster, easier to 
+deploy, and less disruptive way to prevent most users from installing a specific version. If you seek to
+yank a very broken release, you should typically also release a patch release.
+
+There is however, a special category of bugged releases that can not be resolved by having a patch release.
+These also may to be resolved by yanking. That special category is when the compat bounds have been set too
+wide. i.e. say `v2.10.0` was released using a feature not on julia `v1.6` but the compat entry for julia was
+not raised in the release. In this case releasing a `v2.10.1` with the corrected julia compat would not 
+solve the issue as on julia v1.6 Pkg would still resolve the broken `v2.10.0`, and as a minor bump, reverting
+the code changes would not be valid in a patch bump. In this case one may either submit a PR to retroactively
+adjust the compat bounds of previous versions (best user-facing results, but slow and error-prone to implement)
+or yank the offending release.
```

Comment:
We could also link colprac here, which has some detailed guidance: https://github.com/SciML/ColPrac?tab=readme-ov-file#accidental-breaking-releases and https://github.com/SciML/ColPrac?tab=readme-ov-file#accidental-support-for-an-unsupported-dependency

Reply:
```suggestion
If a release is very broken (e.g. contains security vulnerabilities or bugs that are significantly worse 
than the package erroring on load), then it may be yanked. Releasing an ordinary patch release—potentially 
reusing the exact same source code as a previous release without the major bug—is a faster, easier to 
deploy, and less disruptive way to prevent most users from installing a specific version. If you seek to
yank a very broken release, you should typically also release a patch release.

There is however, a special category of bugged releases that can not be resolved by having a patch release.
These also may to be resolved by yanking. That special category is when the compat bounds have been set too
wide. i.e. say `v2.10.0` was released using a feature not on julia `v1.6` but the compat entry for julia was
not raised in the release. In this case releasing a `v2.10.1` with the corrected julia compat would not 
solve the issue as on julia v1.6 Pkg would still resolve the broken `v2.10.0`, and as a minor bump, reverting
the code changes would not be valid in a patch bump. In this case one may either submit a PR to retroactively
adjust the compat bounds of previous versions (best user-facing results, but slow and error-prone to implement)
or yank the offending release. See the [SciML collaborative practices for more guidance](https://github.com/SciML/ColPrac?tab=readme-ov-file#accidental-support-for-an-unsupported-dependency)
```